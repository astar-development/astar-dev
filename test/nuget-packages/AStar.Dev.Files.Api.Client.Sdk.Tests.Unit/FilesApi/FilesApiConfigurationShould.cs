using AStar.Dev.Utilities;

namespace AStar.Dev.Files.Api.Client.Sdk.FilesApi;

public sealed class FilesApiConfigurationShould
{
    [Fact]
    public void ReturnTheExpectedDefaultValue() =>
        new FilesApiConfiguration { Scopes = [] }.ToJson().ShouldMatchApproved();

    [Fact]
    public void ContainTheExpectedProperties() =>
        new FilesApiConfiguration { Scopes = ["MockScope"], BaseUrl = new("https://mock.url.com") }.ToJson().ShouldMatchApproved();

    [Fact]
    public void ReturnTheExpectedSectionLocationValue() =>
        FilesApiConfiguration.SectionLocation.ShouldBe("ApiConfiguration:FilesApiConfiguration");
}
