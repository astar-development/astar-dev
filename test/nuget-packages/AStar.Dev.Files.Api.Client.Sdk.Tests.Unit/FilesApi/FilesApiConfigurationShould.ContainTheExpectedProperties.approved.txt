{
  "baseUrl": "https://mock.url.com",
  "scopes": [
    "MockScope"
  ]
}